qcodes/instrument_drivers/rohde_schwarz/ZNB.py

```diff
@@ -635,15 +643,15 @@ def setup_cw_sweep(self) -> None:
         # set the channel type to single point msmt
         self.sweep_type('CW_Point')
         # turn off average on the VNA since we want single point sweeps.
-        self.write(f'SENS{self._instrument_channel}:AVER:STAT OFF')
+        self.enable_averaging(False)
         # This format is required for getting both real and imaginary parts.
         self.format('Complex')
         # Set the sweep time to auto such that it sets the delay to zero
         # between each point (e.g msmt speed is optimized). Note that if one
         # would like to do a time sweep with time > npts/bandwidth, this is
         # where the delay would be set, but in general we want to measure as
         # fast as possible without artificial delays.
-        self.write(f'SENS{self._instrument_channel}:SWE:TIME:AUTO ON')
```

Comment:
this is not the same as enable averaging right?

Reply:
indeed, it should've been a parameter of its own (and the comment above can move to be part this parameter's docstring)